framework/src/node/state_machine/api_context.ts

```diff
@@ -34,14 +34,29 @@ interface ImmutableSubStoreGetter {
 export const createImmutableAPIContext = (
 	immutableSubstoreGetter: ImmutableSubStoreGetter,
 ): ImmutableAPIContext => ({
-	eventQueue: new EventQueue(),
 	getStore: (moduleID: number, storePrefix: number) =>
 		immutableSubstoreGetter.getStore(moduleID, storePrefix),
 });
 
+export const wrapEventQueue = (eventQueue: EventQueue, topic: Buffer): EventQueueAdder => ({
+	add: (
+		moduleID: number,
+		typeID: Buffer,
+		data: Buffer,
+		topics?: Buffer[],
+		noRevert?: boolean,
```

Comment:
What do you think about a default value here to have a certain false value instead of a falsy value? This can also enable you to change `noRevert` param name to `revert` and practice better naming.

Reply:
I think having default value is fine, but if we do, topics wont be optional or need default value as well.
It is `noRevert` because almost always it should be reverted, and in rare case it should intentionally put to `true`.
That's why it's not `revert` where default is `false`

Reply:
👍